client/src/components/Button/style.css

```diff
@@ -0,0 +1,18 @@
+.button {
```

Comment:
add cursor for the button with a value pointer

Reply:
Done